Dangerfile.swift

```diff
@@ -0,0 +1,12 @@
+// Copyright © 2021 Mindera. All rights reserved.
+
+import Danger
+
+let danger = Danger()
+
+// Swiftlint style check, with comment on PR
+SwiftLint.lint(.all(directory: nil), inline: true)
+
+// List of all edited files
+let editedFiles = danger.git.modifiedFiles + danger.git.createdFiles
+message("These files have changed: \(editedFiles.joined(separator: ", "))")
```

Comment:
Do we need to list files changed? What do we gain with this? Also, we're missing `danger.git.deletedFiles`

Reply:
@diogoeiras I removed this rules (was using for test). Keep only SwiftLint until we have more rules to this specific project.